examples/examples/sloggish/sloggish_subscriber.rs

```diff
@@ -270,6 +270,7 @@ impl Subscriber for SloggishSubscriber {
             comma: false,
         };
         event.record(&mut visitor);
+        #[allow(clippy::write_with_newline)]
         write!(&mut visitor.stderr, "\n").unwrap();
```

Comment:
again, I'd rather just change this to `write_char` or `writeln!(&mut visitor.stderr, "")` rather than adding an `allow` attribute.